assets/javascripts/modules/autoCompleteFactory.js

```diff
@@ -2,15 +2,37 @@ require('jquery');
 var autoComplete = require('./autoComplete.js');
 
 var createAutoCompleteCountries = function () {
-  var $chooseCountryAutoCompleteElem = $('.js-choose-country-auto-complete');
-  var $countryCodeInput = $('#countryCode');
-  var suggestionDisplayTemplate = function (title, value) {
-    return title + ' (+' + value + ')';
-  };
+  $('.js-choose-country-auto-complete').each(function(){
```

Comment:
how are you going to handle multiple autoCompletes with different data? Not sure how wonderful a page with 2 autoCompletes would look! ;) I would call YAGNI on this